### PR TITLE
Is active

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

File: frontend/src/components/collective/CollectiveHero.js [hunk 1]
```diff
@@ -52,7 +52,12 @@ export default class CollectiveHero extends Component {
             </h1>
             <UserAvatarRow members={ collective.members } backers={ collective.backers } otherCount={ collective.contributorsCount } />
             <CollectiveStatsHeader collective={ collective } i18n={ i18n } />
-            <a href='#support' className='mb3 -btn -btn-big -bg-green -ttu -ff-sec -fw-bold'>{ i18n.getString('bePart') }</a>
+            {collective.isActive && 
+              <a href='#support' className='mb3 -btn -btn-big -bg-green -ttu -ff-sec -fw-bold'>{ i18n.getString('bePart') }</a>
+            }
+            {!collective.isActive && host &&
+              <div className="underReview">This collective is currently under review by the host (<a href={host.website}>{host.name}</a>)</div>
```
Comment (asood123) [inline frontend/src/components/collective/CollectiveHero.js]: nice!

File: frontend/src/containers/Collective.js [hunk 2]
```diff
@@ -74,12 +74,14 @@ export class Collective extends Component {
...
       collective,
+      host,
       editCollectiveInProgress,
       cancelEditCollectiveForm,
-      i18n,
-      hasHost
+      i18n
     } = this.props;
 
+    console.log(">>> host: ", host);
```
Comment (asood123) [inline frontend/src/containers/Collective.js]: remove console.log

File: server/src/routes.js [hunk 3]
```diff
@@ -132,6 +133,7 @@ export default (app) => {
   app.get('/:slug([A-Za-z0-9-_]+)/expenses/:expenseid/:action', mw.ga, mw.fetchProfileBySlug, mw.addMeta, render);
   app.get('/:slug([A-Za-z0-9-_]+)/donate/:amount', mw.ga, mw.fetchGroupBySlug, mw.addMeta, render);
   app.get('/:slug([A-Za-z0-9-_]+)/donate/:amount/:interval', mw.ga, mw.fetchGroupBySlug, mw.addMeta, render);
+  app.get('/:slug([A-Za-z0-9-_]+)/apply', mw.ga, mw.fetchProfileBySlug, mw.addMeta, render);
```
Comment (asood123) [inline server/src/routes.js]: why do we need this route and the one above?

File: frontend/src/containers/NewGroup.js [hunk 4]
```diff
@@ -267,13 +273,15 @@ export default connect(mapStateToProps, {
   validateSchema
 })(NewGroup);
 
-export function mapStateToProps({router, form}) {
+export function mapStateToProps({router, form, groups}) {
 
+  const hostCollective = (groups && router.params.slug) ? groups[router.params.slug] : null;
```
Comment (asood123) [inline frontend/src/containers/NewGroup.js]: doesn't this mean that any `/:slug/apply` will work? We should make sure that settings.HostId is valid before this page loads.

File: frontend/src/containers/NewGroup.js [hunk 5]
```diff
@@ -244,6 +249,7 @@ export class NewGroup extends Component {
       mission: attr.mission,
       longDescription: attr.contribute ? `${attr.description}\n\n# Contribute\n\n${attr.contribute}` : attr.description,
       website: attr.website,
+      HostId: hostCollective.settings.HostId,
```
Comment (asood123) [inline frontend/src/containers/NewGroup.js]: where are we setting `isActive` to false when creating this group? Didn't see that on the backend

Comment (xdamman) [inline frontend/src/containers/NewGroup.js]: I've changed the default value of `isActive` to false in the `Group` model in the API.

**Pull request merged**